### PR TITLE
Update Node.js version (using current LTS) for Travis CI

### PR DESCRIPTION
**Pull request by nekonenene**
## Proposed changes

Travis build failed in https://github.com/materializecss/materialize/commit/585245b60a087d9653c61987cabea9c077acd8d2 because of updating `grunt-legacy-log-utils`. We have two choices, downgrade `grunt-legacy-log-utils` or update Node.js, then I choose update Node.js on travis because [6.x supports already ended in April 2019](https://aws.amazon.com/jp/blogs/developer/node-js-6-is-approaching-end-of-life-upgrade-your-aws-lambda-functions-to-the-node-js-10-lts/).

Additionally, remove the unnecessary task installing grunt-cli.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:

- [x] I have read the **[CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md)**.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "6.10"
+  - "12.18.4"
```
Comment (DanielRuf) [inline .travis.yml:3]: Please use the following:

```
- "10"
- "12"
- "latest"
```

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (nekonenene) [reply .travis.yml]: Okay, I did! ("latest" is not good, it causes failed suddenly in the future.)

Comment (Smankusors) [reply .travis.yml]: huh...? I don't think this is what @DanielRuf mean... He mean to use three of them at once on travis.yml? 😅 

IMO it's okay if `latest` is failing, with that we will immediately know that the latest version of Node.js have problem when building materialize.

Comment (DanielRuf) [reply .travis.yml]: "fail early" is what we want, so `latest` should be added.

And `10` too.

Comment (DanielRuf) [reply .travis.yml]: > He mean to use three of them at once on travis.yml?

Correct, all three at once (a build matrix).

> IMO it's okay if latest is failing, with that we will immediately know that the latest version of Node.js have problem when building materialize.

Exactly.

Comment (nekonenene) [reply .travis.yml]: @Smankusors 
I understand now, thanks for your comment! ❤️  Fixed!

Comment (DanielRuf) [reply .travis.yml]: It seems nvm doesn't have this tag anymore. Can you try `node` and `*` instead of `latest`?

Comment (nekonenene) [reply .travis.yml]: @DanielRuf 
Thank you! Yes, I just did it and succeeded!

**Review by DanielRuf — APPROVED** [review]

**Pull request merged**